charts/backingservices/requirements.yaml

```diff
@@ -1,6 +1,7 @@
-# Pega does NOT actively update this elasticsearch dependency. To leverage SRS, you should:
-# * Update the elasticsearch 'version' value (the version value should be same as imageTag in values.yaml), or
-# * Disable deploying elasticsearch by setting 'srs.srsStorage.provisionInternalESCluster' to false.
+# Pega does not actively update this Elasticsearch dependency. To use the SRS to provide an Elasticsearch service to your deployment you must:
+# * Update the Elasticsearch helm chart 'version' value (the version value should be same as imageTag in values.yaml).
+# * For kubernetes version >=1.25 or Elasticsearch image version 7.17.9 use Elasticsearch helm chart version 7.17.3, or
+# * Disable deploying Elasticsearch in SRS by setting 'srs.srsStorage.provisionInternalESCluster' in backingservices values.yaml to false.
```

Comment:
We're not using the term `Embedded elasticsearch` anywhere. we're calling it as an `internally-provisioned elasticsearch` everywhere.
Please correct me if I'm wrong here. @taz-pega-work

Reply:
I think we're confusing two different things here - the internally-provisioned Elasticsearch in the requirements.yaml file is the open source cluster you can deploy as storage for SRS (instead of an existing externally managed Elasticsearch), NOT the embedded Elasticsearch/Elasticsearch plug-in that we're deprecating.

Reply:
Can we connect once on this @kingakowalska1 / @taz-pega-work ?